### DISCUSSION TITLE
Error when upgrading to ios 18

### DISCUSSION THREAD

**Discussion by Rezanoor** [groue/GRDB.swift#1669] [General]
I'm getting an error when upgrading to ios 18. attached is the screen shot of the error. Any idea?
![Screenshot 2024-11-20 at 2 33 42 pm](https://github.com/user-attachments/assets/3015dda2-6804-4731-869c-b30407842278)

**Comment by groue** [comment]
Hello @Rezanoor,

This might by #1619, which was fixed in Xcode 16.1.

If not, you did not provide enough information: lease open a [new issue](https://github.com/groue/GRDB.swift/issues/new) and take the time to fill in the template, thank you.
